### ISSUE TITLE
Booting up the game crashes it.

### ISSUE THREAD

**Issue by Fanrail** [ValkyRiver/upgrademod-wip#2]
And I don't even know what may cause it, because the BSOD message did not specify any mod:
Oops! The game crashed:
Syntax error: card.lua:2682: unexpected symbol near ')'

Additional Context:
Balatro Version: 1.0.1n-FULL (Локализация от Nitablade, v1.1.6)
Modded Version: 1.0.0~ALPHA-1229b-STEAMODDED
LÖVE Version: 11.5.0
Lovely Version: 0.6.0

Stack Traceback
===============
(3)  C function 'function: 0x3260bbe8'
(4) global C function 'require'
(5) main chunk of file 'main.lua' at line 881
(6) global C function 'require'
(7) LÖVE function at file 'boot.lua:323' (best guess)
Local variables:
 c = table: 0x325fe6a0  {identity:false, version:11.5, accelerometerjoystick:true, modules:table: 0x3260a668 (more...)}
 openedconsole = boolean: false
 confok = boolean: true
 conferr = nil
(8) global C function 'xpcall'
(9) LÖVE function at file 'boot.lua:362' (best guess)
Local variables:
 result = boolean: true
(10) global C function 'xpcall'
(11) LÖVE function at file 'boot.lua:377' (best guess)
Local variables:
 func = Lua function '(LÖVE Function)' (defined at line 355 of chunk [love "boot.lua"])
 inerror = boolean: true
 deferErrhand = Lua function '(LÖVE Function)' (defined at line 348 of chunk [love "boot.lua"])
 earlyinit = Lua function '(LÖVE Function)' (defined at line 355 of chunk [love "boot.lua"])

**Comment by ValkyRiver** [comment]
Can you check if you have any other mods installed? This mod is incompatible with most other mods.
Also, the no-Talisman version is outdated, and the Talisman version works even if Talisman is not installed.

**Comment by Fanrail** [comment]
Yes, I do have many other mods, and do not have Talisman, and used the Talisman version. Now, after opening the game with this new version while having all other mods disabled, I get this death screen message:

Oops! The game crashed:
Syntax error: card.lua:2682: unexpected symbol near ')'


Additional Context:
Balatro Version: 1.0.1n-FULL
Modded Version: 1.0.0~ALPHA-1229b-STEAMODDED
LÖVE Version: 11.5.0
Lovely Version: 0.6.0

Stack Traceback
===============
(3)  C function 'function: 0x10ebcef0'
(4) global C function 'require'
(5) main chunk of file 'main.lua' at line 881
(6) global C function 'require'
(7) LÖVE function at file 'boot.lua:323' (best guess)
Local variables:
 c = table: 0x10ebf330  {identity:false, version:11.5, accelerometerjoystick:true, modules:table: 0x10ebf380 (more...)}
 openedconsole = boolean: false
 confok = boolean: true
 conferr = nil
(8) global C function 'xpcall'
(9) LÖVE function at file 'boot.lua:362' (best guess)
Local variables:
 result = boolean: true
(10) global C function 'xpcall'
(11) LÖVE function at file 'boot.lua:377' (best guess)
Local variables:
 func = Lua function '(LÖVE Function)' (defined at line 355 of chunk [love "boot.lua"])
 inerror = boolean: true
 deferErrhand = Lua function '(LÖVE Function)' (defined at line 348 of chunk [love "boot.lua"])
 earlyinit = Lua function '(LÖVE Function)' (defined at line 355 of chunk [love "boot.lua"])
